packages/react-native-editor/package.json

```diff
@@ -1,150 +1,49 @@
 {
   "name": "@wordpress/react-native-editor",
   "version": "1.20.0",
+  "description": "Mobile WordPress gutenberg editor.",
+  "author": "The WordPress Contributors",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "wordpress",
+    "react-native"
+  ],
   "private": true,
   "config": {
     "jsfiles": "./*.js src/*.js src/**/*.js src/**/**/*.js",
     "scssfiles": "src/*.scss src/**/*.scss"
   },
+  "homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/react-native-editor/README.md",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/WordPress/gutenberg.git",
+    "directory": "packages/react-native-editor"
+  },
+  "bugs": {
+    "url": "https://github.com/WordPress/gutenberg/issues"
+  },
   "engines": {
     "node": ">=8.0.0 <9.0.0 || >=10.0.0",
     "npm": ">=6.0.0"
   },
-  "devDependencies": {
-    "@babel/core": "^7.3.3",
-    "@wordpress/babel-preset-default": "^4.0.0",
-    "@wordpress/eslint-plugin": "^2.0.0",
-    "@wordpress/jest-preset-default": "^4.0.0",
-    "appium": "1.16.0-rc.1",
-    "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^8.2.2",
-    "babel-jest": "^24.1.0",
-    "babel-plugin-react-native-classname-to-style": "^1.2.1",
-    "babel-plugin-react-native-platform-specific-extensions": "^1.1.1",
-    "cross-env": "^5.1.4",
-    "empty": "^0.10.1",
-    "enzyme": "^3.9.0",
-    "enzyme-adapter-react-16": "^1.11.2",
-    "enzyme-to-json": "^3.3.5",
-    "equivalent-key-map": "^0.2.0",
-    "eslint": "^5.12.1",
-    "eslint-plugin-flowtype": "^3.4.2",
-    "eslint-plugin-jest": "^21.15.0",
-    "eslint-plugin-react-native": "^3.6.0",
-    "eslint-plugin-wordpress": "git://github.com/WordPress-Coding-Standards/eslint-plugin-wordpress.git#552af1454d175e15f6d25aadc2ccde30a1922d4f",
-    "flow-bin": "0.92.0",
-    "husky": "^3.0.2",
-    "identity-obj-proxy": "^3.0.0",
-    "jest": "^24.1.0",
-    "jest-junit": "^6.3.0",
-    "metro-react-native-babel-preset": "^0.52.0",
-    "node-fetch": "^2.3.0",
-    "prettier": "git+https://github.com/Automattic/calypso-prettier.git#calypso-1.9",
-    "prettier-eslint": "^8.8.2",
-    "prettier-eslint-cli": "^4.7.1",
-    "react-dom": "16.8.6",
-    "react-native-sass-transformer": "^1.1.1",
-    "react-test-renderer": "16.8.3",
-    "regenerator-runtime": "^0.12.1",
-    "remote-redux-devtools": "^0.5.12",
-    "rungen": "^0.3.2",
-    "sprintf-js": "^1.1.1",
-    "wd": "^1.11.1"
-  },
   "husky": {
```

Comment:
Prettier is configured for the whole repository so you should remove husky integration here.